tools/config/seed-advanced.config.ts

```diff
@@ -1,11 +1,30 @@
 import { argv } from 'yargs';
 import { SeedConfig } from './seed.config';
+import * as path from 'path';
 import { ExtendPackages } from './seed.config.interfaces';
 
 export class SeedAdvancedConfig extends SeedConfig {
+  /**
+   * The base folder of the nativescript applications source files.
+   * @type {string}
+   */
+  TNS_BASE_DIR = 'nativescript';
+
+  srcSubdir = 'src';
+  destSubdir = 'app';
+
+  TNS_APP_SRC = `${this.TNS_BASE_DIR}/${this.srcSubdir}`;
+
+  TNS_APP_DEST = `${this.TNS_BASE_DIR}/${this.destSubdir}`;
+
+  TNS_CONFIG = {
+    ANALYTICS_TRACKING_ID: '',
+  };
 
   constructor() {
     super();
+    this.ENABLE_SCSS = true;
```

Comment:
Should SCSS be enabled? I think this line got copied by accident from our project.